package.json

```diff
@@ -1,19 +1,20 @@
 {
-  "name": "git-butler-tauri",
-  "private": true,
-  "scripts": {
-    "dev": "pnpm --filter @gitbutler/ui run dev",
-    "test": "pnpm --filter @gitbutler/ui run test",
-    "build": "pnpm --filter @gitbutler/ui run build",
-    "build:nightly": "pnpm --filter @gitbutler/ui run build:nightly",
-    "build:development": "pnpm --filter @gitbutler/ui run build:development",
-    "check": "pnpm --filter @gitbutler/ui run check",
-    "lint": "pnpm --filter @gitbutler/ui run lint",
-    "format": "pnpm --filter @gitbutler/ui run format",
-    "tauri": "tauri",
-    "prepare": "pnpm --filter @gitbutler/ui run prepare"
-  },
-  "devDependencies": {
-    "@tauri-apps/cli": "^1.5.10"
-  }
+	"name": "git-butler-tauri",
+	"private": true,
+	"scripts": {
+		"dev": "pnpm --filter @gitbutler/ui run dev",
+		"test": "pnpm --filter @gitbutler/ui run test",
+		"test:watch": "pnpm --filter @gitbutler/ui run test:watch",
+		"build": "pnpm --filter @gitbutler/ui run build",
+		"build:nightly": "pnpm --filter @gitbutler/ui run build:nightly",
+		"build:development": "pnpm --filter @gitbutler/ui run build:development",
+		"check": "pnpm --filter @gitbutler/ui run check",
+		"lint": "pnpm --filter @gitbutler/ui run lint",
+		"format": "pnpm --filter @gitbutler/ui run format",
+		"tauri": "tauri",
+		"prepare": "pnpm --filter @gitbutler/ui run prepare"
+	},
+	"devDependencies": {
+		"@tauri-apps/cli": "^1.5.10"
+	}
```

Comment:
Side note, I wonder if we don't have a formatting specified for json in ci since it doesn't complain? We should perhaps keep the original unless we're now doing tabs.

Reply:
The tauri.conf.*.json files are all tabs and the gitbutler-ui/package.json is also tabs, but, I'd be happy to put that package.json back if we're not ready to make a particular decision on that.

Reply:
You're right, we should prefer consistency with the other json files. :)